src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -99,10 +99,14 @@ static bool mpu6500Detect(void)
     uint8_t tmp;
 
     mpu6500ReadRegister(MPU6500_RA_WHOAMI, &tmp, 1);
+#ifdef COLIBRI_RACE
```

Comment:
It should be returning something else, but i'm haven't check it yet.
I have read the datasheet, the WHOAMI for that chip should be right, but still returning failed on me.
Did anyone try this code and sure it is working?